runtime/j9vm/jvm.c

```diff
@@ -4004,6 +4015,32 @@ JVM_LoadLibrary(const char *libName, jboolean throwOnFailure)
 	return result;
 }
 
+/* NOTE this is required by JDK15+ jdk.internal.loader.NativeLibraries.unload().
+ */
+#if JAVA_SPEC_VERSION >= 15
+void JNICALL JVM_UnloadLibrary(void *handle)
+#else /* JAVA_SPEC_VERSION >= 15 */
+jobject JNICALL JVM_UnloadLibrary(jint arg0)
+#endif /* JAVA_SPEC_VERSION >= 15 */
+{
+#if JAVA_SPEC_VERSION >= 15
+	Trc_SC_UnloadLibrary_Entry(handle);
```

Comment:
Please add and use a matching 'exit' tracepoint for this function.

Reply:
@keithc-ca / @pshipton : Quick q... should I be adding the exit tracepoint right after the existing `Trc_SC_UnloadLibrary_Entry` in `runtime/j9vm/j9scar.tdf`, or should it be added to the end to not mess up the existing indices?

Reply:
It needs to be added at the end.